checkout.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet"  type="text/css" href="styles/stylesheet.css">
+    <title>Adopt A Dog</title>
+  </head>
+  <body>
+    <a href="#main-content" class="hidden">Skip to main content</a>
+    <header>
+      <div class="head-container">
+        <section id="id-img">
+          <img src="images/adoptadog-logo.svg" alt="Company dog logo">
+          <h1>Seattle's premiere dog adoption agency</h1>
+        </section>
+        <section class="cart-holder">
+          <h6>Shopping Cart</h6>
+          <img id="cart-image" src="images/cart-icon.svg" alt="shopping cart">
+          <p id="sum">$1234.56</p> <!--This is a placeholder-->
+          <a class="checkout-button" href="checkout.html">Checkout</a>
+        </section>
+      </div>
+    </header>
+    <nav class="top-nav">
+      <div class="standard-width">
+        <a href="index.html">Home</a> &nbsp;&nbsp;/&nbsp;&nbsp; <a href="dogs.html">Dogs</a> &nbsp;&nbsp;/&nbsp;&nbsp; <a href="blog.html">Blog</a>
+      </div>
+    </nav>
+    <section id="main-content" class="standard-width top-spacer">
+      <h2 class="occupy-solo">Checkout</h2>
+      <form method="GET" class="occupy-solo">
```

Comment:
Id suggest modifying the form fields font to better match the size and style of the page.